CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+### Unfixed
+- Remove auto scroll from add story method
+- Add highlight function for when a story be saved
+- Refactor highlightSearchedStories method
```

Comment:
I doesn't need to be declared here.

Just log the effect changes which describes a feature behavior, not the code detail change.